src/data/rat-partner.json

```diff
@@ -0,0 +1,89 @@
+{
+    "section-main": {
+        "headline": {
+            "title": "Become a rapid test partner!",
+            "textblock": [
+                "Welcome to the Corona-Warn-App-Community!",
```

Comment:
So you consider RAT partners as community?

Reply:
Well, I just used the text that was shown in the mentioned issue. I can understand your concern with this specific line, I'd suggest to just remove this line. What do you think?

Reply:
Yes, this sounds good to me! Thanks!